### PR TITLE
NAS-132743 / 25.04 / Fix cloud backup tests

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

**Comment by themylogin** [comment]
time 0:30

File: src/middlewared/middlewared/api/v25_04_0/cloud_backup.py [hunk 1]
```diff
@@ -31,8 +31,8 @@ class CloudBackupCreate(BaseModel):
     pre_script: LongString = ""
     post_script: LongString = ""
     snapshot: bool = False
-    include: list[NonEmptyString]
-    exclude: list[NonEmptyString]
+    include: list[NonEmptyString] = []
```
Comment (yocalebo) [inline src/middlewared/middlewared/api/v25_04_0/cloud_backup.py:34]: Should the type annotation be `list | list[NonEmptyString] = []`?

Comment (themylogin) [reply src/middlewared/middlewared/api/v25_04_0/cloud_backup.py:34]: An empty list is a valid list of non-empty strings. Non-empty list of non-empty strings would be `Annotated[list[NonEmptyString], Len(min_length=1)]`

Comment (yocalebo) [reply src/middlewared/middlewared/api/v25_04_0/cloud_backup.py:34]: That doesn't make logical sense. With that logic, this means this is equal `['a']` == `[]`

Comment (yocalebo) [reply src/middlewared/middlewared/api/v25_04_0/cloud_backup.py:34]: I used `mypy` to statically analyze the annotation and it didn't complain using your annotation or mine so I guess we'll leave this as is but, for the record, claiming `list[NonEmptyString]` is equivalent to an empty list is really strange and not logical imo.

**Review by yocalebo — APPROVED** [review]

**Review by creatorcary — APPROVED** [review]

**Pull request merged**